pandas/tests/reshape/merge/test_merge.py

```diff
@@ -410,6 +410,39 @@ def test_merge_non_unique_index_many_to_many(self):
         )
         _check_merge(df1, df2)
 
+    def test_merge_right_duplicate_suffix(self):
+        # https://github.com/pandas-dev/pandas/issues/29697
+
```

Comment:
could u add one or more sentences of comments here to address your testing purpose?

Reply:
It's just a simple test case directly addressing the issue which I linked. I think that should suffice.

Reply:
Okay make sense